### PR TITLE
Suggested changes and fixes

### LINKED ISSUE #15: Suggestions
Would suggest changing

PadlockAPI#isMember(@NotNull Block block, @NotNull Player player)

to 

PadlockAPI#isMember(@NotNull Block block, @NotNull OfflinePlayer offlinePlayer)

which would allow plugins that might hook into Padlock to check if offline players can access specific block.

---------------

It would probably be good idea to add some more aliases from similar plugins, just so it makes life a bit easier for players:

aliases: [ lock, padlock, lockit, lockette, deadbolt ]

---------------

Updating signs, not sure how signs are currently updated but from what I could see when testing, usernames on signs are not automatically updated? I'd suggest maybe adding feature where the sign would be updated on interact if player is online and has clicked on outdated sign?

### PR DESCRIPTION
**Pull request by FireInstall**
closes #15

- Rename Command to MainCommand to better differentiate between it and Spigots Command class.
- Removed (Offline)Player from the API to work with UUIDs instead to allow offline player check without depending on the OfflinePlayer class since it may change in the future 
- Nag Admins to remove LockettePro instead of informing them it was missing.
- Unregister LockettePro commands. Aliases seem like they are broken even though we are softdepending on LockettePro and should loading after. Should be solveable if we would switch to paper plugins. However since Lockette should get removed from plugin list anyway it shouldn't be a big problem.
- Fix broken config enums
- Fix buckets

### CODE REVIEW

**Comment by FireInstall** [comment]
@IAISI have a look this should be everything you mentioned in the issue

File: src/main/resources/lang.properties [hunk 1]
```diff
@@ -30,7 +30,8 @@ cmd.help.set-access-type=/lock setaccesstype <private/public/donation/supply/dis
 cmd.help.set-connected=/lock setconnected <true/false> | set if adjacent (trap)doors / fence should open/close together
 cmd.help.set-password=/lock setpassword <password> you may need to get access via /lock password first.
 cmd.help.set-timer=/lock settimer 0t 0s 0m 0h| set the timer a door/gate/trapdoor should toggle after used in ticks, seconds, minutes, or hours. Note: does not persist when server restarts.
-cmd.help.update-sign=/lock updatesign | manuel start a updating process for a legacy lockette sign.
+cmd.help.update-display=/lock updatedisplay | manuel update the display of a sign.
```
Comment (Schpammer) [inline src/main/resources/lang.properties]: "manually updates". Same in line 34 and other files with default English strings

Comment (FireInstall) [reply src/main/resources/lang.properties]: Done

**Comment by FireInstall** [comment]
Rebase onto main

**Comment by NO-UUID** [comment]
Sorry for late reply, been super busy lately, but yea this looks good :)

**Comment by FireInstall** [comment]
@Schpammer anything else?

**Review by Schpammer — APPROVED** [review]

**Pull request merged**